File: fsharp/card-game-war/test/game-test.fs
module game_test

open Expecto
open card_game_war.game

// fill in  tests for your game

[<Tests>]
let tests =
    testList "game" [
        testCase "the highest rank wins the cards in the round" <| fun _ ->
            Expect.equal
                0 1 ""
        testCase "queens are higher rank than jacks" <| fun _ -> ()
        testCase "kings are higher rank than queens" <| fun _ -> ()
        testCase "aces are higher rank than kings" <| fun _ -> ()

        testCase "the player loses when they run out of cards" <| fun _ -> ()
    ]

[<EntryPoint>]
let main argv =
    runTestsInAssembly defaultConfig argv

File: csharp/card-game-war/test/game-test.cs
namespace card_game_war;

using Xunit;
using static card_game_war.game;

public class game_test {
    [Fact]
    public void highest_rank_wins_cards_in_round() {
        Assert.Equal (
            0, 1
        );
    }

    [Fact]
    public void queens_higher_rank_than_jacks() {}

    [Fact]
    public void kings_higher_rank_than_queens() {}

    [Fact]
    public void aces_higher_rank_than_kings() {}

    [Fact]
    public void player_loses_when_out_of_cards() {}
}
